specification/liftrastronomer/Astronomer.Astro.Management/LiftrBase/main.tsp

```diff
@@ -65,6 +69,10 @@ model OfferDetails {
 
   @doc("Plan Display Name for the marketplace offer")
   termId?: string;
+
+  @added(Versions.v2024_08_27_preview)
+  @doc("Subscription auto renewal flag")
+  autoRenew?: boolean;
```

Comment:
>autoRenew [](http://example.com/codeflow?start=2&length=9)

how about a property "renewalMode" and values can be enums of "Auto". "Fixed" or "Default" etc.

Reply:
We need to pass true/false for this property to Saas RP Endpoint.
![Screenshot 2024-08-28 110513](https://github.com/user-attachments/assets/8cd6d762-00e3-42c8-adb9-89dab3f6f878)